FLT/for_mathlib/Frobenius.lean

```diff
@@ -415,7 +415,7 @@ theorem pow_eq_expand (a : (A ⧸ P)[X]) :
   have pprime : Nat.Prime p := p_is_prime P
   have factprime : Fact (Nat.Prime p) := { out := pprime }
   rw [← q_is_p_pow_n, ← map_expand_pow_char, map_expand, FiniteField.frobenius_pow]
-  · simp only [exists_and_left, RingHom.one_def, map_id]
+  · simp [exists_and_left, RingHom.one_def, map_id]
```

Comment:
Any particular reason to use `simp` rather than `simp only`? You'd probably want to remove the lemmas in the default simp set from the explicit list of arguments here.

Reply:
Please don't worry about these files. They are all dead code. I want to keep them for reference only. How should I indicate this? The theory of Frobenius elements needs to be developed in mathlib following the roadmap at https://leanprover.zulipchat.com/#narrow/stream/416277-FLT/topic/Outstanding.20Tasks.2C.20V4/near/449562398 .

Reply:
@kbuzzard Oh I see. I didn’t know that, sorry. An idea might be to add a comment at the beginning and then .gitignore it.